interop/src/client.rs

```diff
@@ -290,3 +291,265 @@ impl Client {
         Ok(())
     }
 }
+
+
+#[cfg(feature = "protobufv3-codec")]
```

Comment:
It looks like you could move this conditional down to the level of the methods — `new` looks like it's identical in the two branches.

Reply:
I thought it would be cleaner by doing it at the Impl level, but you're right, this duplicates some code and adds stuff to maintain. Done, thanks